sdk/formrecognizer/Azure.AI.FormRecognizer/src/BoundingBox.cs

```diff
@@ -18,16 +18,18 @@ namespace Azure.AI.FormRecognizer.Models
     {
         internal BoundingBox(IReadOnlyList<float> boundingBox)
         {
-            // TODO: improve perf here?
-            // https://github.com/Azure/azure-sdk-for-net/issues/10358
-            float[] bbPoints = boundingBox.ToArray();
+            if (boundingBox.Count == 0)
+            {
+                Points = null;
```

Comment:
>This is not a scenario we're expecting to happen, but wouldn't it make more sense to make Points an empty array? I prefer throwing an IndexOutOfRange than a NullReference, since we're making lists empty across the client when the service doesn't return anything.

I just brought the original behavior back so the tests pass as is. Up to you if you want something changed. I think not having it be null is a good idea.